docs/afxdp.md

```diff
@@ -0,0 +1,23 @@
+# AF_XDP interface
+
+This header declares the AF_XDP sockets interface. AF_XDP sockets are used by user mode applications to receive/inspect/drop/send network traffic via XDP hook points. To receive traffic, packets must be steered to a socket by configuring XDP rules/programs using an interface declared elsewhere. Traffic is passed as flat buffer, L2 frames across this interface using single producer, single consumer shared memory rings.
+
+> **TODO** Complete AF_XDP documentation.
+
+## See Also
```

Comment:
I see all other docs are dropped in the `api` directory. What do you say to splitting that out, instead having an `afxdp` directory for the UM API, and then putting the other related functions and types in a different relevant dir?

Reply:
Yeah I've wondered about organization. Part of the issue is our long term goal is to share types across various XDP layers, so the same offload extensions get used by both native driver and AF_XDP, for example. I agree we have tons of docs here and some kind of grouping or sorting would be useful.

Reply:
Yeah, I definitely think it would be useful. Ideally adding some split now would be good, but I won't block the PR on it if you want to do it in a follow up.

Reply:
The `See Also` section of `afxdp.md` was an attempt at providing that kind of index, since each file can belong to multiple API families by design.